src/promptflow-core/promptflow/executor/_tool_resolver.py

```diff
@@ -527,6 +527,8 @@ def _resolve_llm_connection_with_provider(connection):
         return connection, provider
 
     def _resolve_llm_node(self, node: Node, convert_input_types=False) -> ResolvedTool:
+        logging.warning("Please upgrade to the latest version of promptflow-tools to consume new LLM tools. "
+                        "You can upgrade by running 'pip install promptflow-tools --upgrade'")
```

Comment:
difference betrween logger and module_logger? Shall I use module_logger or anything else?
![Uploading image.png…]()